src/trinerdi/template.cpp

```diff
@@ -0,0 +1,15 @@
+#include <bits/stdc++.h>
+
+using namespace std;
+
+typedef long long ll;
+typedef long double ld;
+#define rep(i, a, n) for (int i = (a); i < (n); i++)
+#define per(i, a, n) for (int i = (n) - 1; i >= (a); i--)
+
+int n;
```

Comment:
Should `n` be global here?

Reply:
On a second thought, it probably shouldn't be here at all :slightly_smiling_face: I've found that usually I delete the `cin >> n` line anyway, if only because of renaming the variable.